### PR TITLE
Docs update 2

### PR DESCRIPTION
**Pull request by jthandy**

### CODE REVIEW

**Comment by jthandy** [comment]
for https://github.com/analyst-collective/dbt/issues/141

**Comment by jthandy** [comment]
@drewbanin view development docs here:
http://dbt.readthedocs.io/en/development/

readthedocs will automatically build your changes to master docs when development is merged into master. nice!

also: you should sign up for a readthedocs user so that i can add you as a maintainer.

File: docs/guide/configuring-models.md [hunk 1]
```diff
@@ -16,49 +15,88 @@ Here is how these configuration options look in practice:
...
 
-Incremental models are a powerful feature in production dbt deployments. Frequently, certain raw data tables can have billions (or more) rows, which makes performing frequent rebuilds of models dependent on these tables impractical. Incremental tables provide another option. The first time a model is deployed, data is populated in a `CREATE TABLE AS SELECT...` statement, but in subsequent runs this model will have incremental rows populated via an `INSERT` statement wrapped around the underlying `SELECT`.
+#### using {{this}}
 
-The key to making incremental models work is supplying a `WHERE` condition that identifies the rows that need to be added. For instance, in a clickstream table, you might apply the condition `WHERE [source].event_date > [model].event_date`. Practically, dbt applies this where condition for you, you just need to specify the field name to apply the condition on via the `incremental-id` parameter. All incremental tables require this parameter.
+`{{this}}` is a special variable that returns the schema and table name of the current model and is useful when defining a `sql_where` clause. The `sql_where` we wrote earlier would actually be written as such:
```
Comment (drewbanin) [inline docs/guide/configuring-models.md]: I added a new feature in the hooks branch -- now you can do:
```
{{this}} ==> "schema"."table"
{{this.schema}} ==> schema
{{this.table}} ==> table
```

File: sample.dbt_project.yml [hunk 2]
```diff
@@ -14,48 +15,46 @@ test-paths: ["test"]         # where to store test results
...
-# profile: my-org            # use my-org profile within this project. uncomment to enable
-
-# default paramaters that apply to _all_ models (unless overridden below)
+# configuration of default database information, as defined within ~/.dbt/profiles.yml.
+# all execution of dbt from the root directory of this project will default to this profile.
 
-model-defaults:
-  enabled: true           # enable all models by default
-  materialized: view      # options: table | view | ephemeral | incremental
+profile: evil-corp           # use my-org profile within this project.
```
Comment (drewbanin) [inline sample.dbt_project.yml]: you dawgg

Comment (drewbanin) [inline sample.dbt_project.yml]: but also we should change `my-org` to `evil-corp` in this example

File: docs/guide/best-practices.md [hunk 3]
```diff
@@ -1,40 +1,35 @@
...
 
 1. Use views instead of tables to the greatest extent possible in development. Views typically deploy much faster than tables, and in development it's often not critical that subsequent analytic queries run as fast as possible. It's easy to change this setting later and it will have no impact on your business logic.
 1. Use `dbt_project.yml` to disable portions of your project that you're not currently working on. If you have multiple modules within a given project, turn off the ones that you're not currently working on so that those models don't deploy with every `dbt run`.
-1. Pass the `--model` flag to `dbt run`. This flag asks dbt to only `run` the models you specify and their dependents. If you're working on a particular model, this can make a very significant difference in your workflow.
+1. Pass the `--models` flag to `dbt run`. This flag causes dbt to only deploy the models you specify and their dependents. If you're working on a particular model, this can make a very significant difference in your deployment time in large projects.
+1. [mention limitation based on `target` once that is built]
```
Comment (jthandy) [inline docs/guide/best-practices.md]: 👍 

we have an issue for this, added a note to it:
https://github.com/analyst-collective/dbt/issues/149

File: docs/guide/configuring-models.md (hunk 1 above)
Comment (jthandy) [inline docs/guide/configuring-models.md]: added.

👍

**Comment by jthandy** [comment]
@drewbanin thanks for comments! addressed everything, merging to develop now.

**Pull request merged**